api/cometbft/mempool/v1/message.go

```diff
@@ -13,12 +13,28 @@ func (m *Txs) Wrap() proto.Message {
 	return mm
 }
 
+func (m *HaveTx) Wrap() proto.Message {
```

Comment:
Is the plan to have DOG in `v1`?

Reply:
Not when we are so close to release `v1` but it's possible that will land in `v1.x-experimental`.

Reply:
Then maybe we should create proto `v2`?

Reply:
There's definitely material for `v2` in `main`.

Reply:
Should we do it here or open another PR?

Reply:
Sorry, I misunderstood the original question. I thought you were asking about Comet's version. 

I agree we should update the mempool proto version to `v2`. Will do it here in this PR.

Reply:
Done.